tsdataclinic/scout
# Issue #373

## Summary

Add a new field (city) into Dataframe so that we can tell which city the current dataset is inside MapViz. Change index.tsx and socrataResultsToDataframe.ts files accordingly so that we can get the city abbreviation. Add a json file to store location of each city (for now, it only has Chicago and New York).

## Screenshots or Videos (if applicable)
Example of what it looks like when click on map under visualize tab for a dataset in Chicago.

![image](https://github.com/tsdataclinic/scout/assets/103480378/93068fd6-ca8a-4629-a84c-afdd3e004c06)


Example of what it looks like whne click on map under visualize tab for a dataset in New York.

![image](https://github.com/tsdataclinic/scout/assets/103480378/7566c52c-f1bd-4776-a6e1-902e0a1bac4b)


## Related Issues

N/A

## Test Plan

1.  Go to Explore
2.  Choose a city (Chicago or New York)
3.  Click on any dataset
4.  Click on Visualize tab
5.  Change to map
6. It should zoom in whichever city of the dataset that you choose on step 2 (only works for Chicago and New York for now)

## Checklist Before Requesting a Review

- [ ] My code follows the Style Guidelines and Best Practices outlined in the project wiki
- [ ] My change generates no new warnings or failed tests
